ocs_ci/ocs/uninstall.py

```diff
@@ -228,41 +313,11 @@ def uninstall_ocs():
         "ocsinitializations.ocs.openshift.io",
         "storageclusterinitializations.ocs.openshift.io",
```

Comment:
storageclusterinitializations.ocs.openshift.io has to be removed from the list as per ocs 4.6 . Ref : https://github.com/red-hat-storage/ocs-ci/issues/3356